sentry/src/main/java/io/sentry/transport/AsyncConnection.java

```diff
@@ -62,6 +63,7 @@ public AsyncConnection(
     this.executor = executorService;
   }
 
+  @SuppressWarnings("SyntheticAccessor")
```

Comment:
inner classes have access to private fields but this has side effects, see https://stackoverflow.com/questions/4501324/synthetic-accessor-method-warning
so either we create a getter which cant be private, nor protected as the class is final, but I'd not like to make it public, to passing the instance is the right choice